Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM java:8
+
+ENV SPRING_PROFILES_ACTIVE production
+
+EXPOSE 8080
+EXPOSE 8081
+
+CMD java -jar /data/app.jar
+
+ADD build/libs/identity-service.jar /data/app.jar
+
+ADD https://github.com/Civil-Service-Human-Resources/lpg-terraform-paas/releases/download/hammer-0.1/hammer /bin/hammer
```

Comment:
INFO - Do we need to add or pass ENV variables?

Reply:
We can pass any env vars - they don't need to be defined in the Dockerfile. Normally I would define them if we need/want to specify a default for the container, like with the spring profile. But mostly our defaults will come from the application.properties in the code.